packages/cli/src/transforms/toktx.ts

```diff
@@ -154,33 +160,46 @@ export const toktx = function (options: ETC1SOptions | UASTCOptions): Transform
 				const outPath = tmp.tmpNameSync({ postfix: '.ktx2' });
 
 				const inBytes = image.byteLength;
-				fs.writeFileSync(inPath, Buffer.from(image));
+				await fs.writeFile(inPath, Buffer.from(image));
 
-				const params = [...createParams(slots, channels, size, logger, options), outPath, inPath];
+				const params = [...createParams(slots, channels, size, logger, numTextures, options.jobs!, options), outPath, inPath];
```

Comment:
Nit: jobs is part of the options object, so no need to duplicate it here.

```suggestion
				const params = [...createParams(slots, channels, size, logger, numTextures, options), outPath, inPath];
```